src/pages/Embark/Landing.tsx

```diff
@@ -147,26 +149,61 @@ export const Landing: React.FC<{ language?: string }> = ({ language }) => {
 
         <Wrapper>
           <Container>
-            <Card>
-              <Headline>{textKeys.STARTPAGE_UNINSURED_HEADLINE()}</Headline>
-              <Paragraph>{textKeys.STARTPAGE_UNINSURED_BODY()}</Paragraph>
-              <ProceedButton
-                size="lg"
-                to={(language ? '/' + language : '') + '/new-member/new'}
-              >
-                {textKeys.STARTPAGE_UNINSURED_BUTTON()}
-              </ProceedButton>
-            </Card>
-            <Card>
-              <Headline>{textKeys.STARTPAGE_INSURED_HEADLINE()}</Headline>
-              <Paragraph>{textKeys.STARTPAGE_INSURED_BODY()}</Paragraph>
-              <ProceedButton
-                size="lg"
-                to={(language ? '/' + language : '') + '/new-member/switch'}
-              >
-                {textKeys.STARTPAGE_INSURED_BUTTON()}
-              </ProceedButton>
-            </Card>
+            {market === Market.Se && (
+              <>
+                <Card>
+                  <Headline>{textKeys.STARTPAGE_UNINSURED_HEADLINE()}</Headline>
+                  <Paragraph>{textKeys.STARTPAGE_UNINSURED_BODY()}</Paragraph>
+                  <ProceedButton
+                    size="lg"
+                    to={(language ? '/' + language : '') + '/new-member/new'}
+                  >
+                    {textKeys.STARTPAGE_UNINSURED_BUTTON()}
+                  </ProceedButton>
+                </Card>
+                <Card>
+                  <Headline>{textKeys.STARTPAGE_INSURED_HEADLINE()}</Headline>
+                  <Paragraph>{textKeys.STARTPAGE_INSURED_BODY()}</Paragraph>
+                  <ProceedButton
+                    size="lg"
+                    to={(language ? '/' + language : '') + '/new-member/switch'}
+                  >
+                    {textKeys.STARTPAGE_INSURED_BUTTON()}
+                  </ProceedButton>
+                </Card>
+              </>
+            )}
```

Comment:
What about moving this stuff to separate components, like a `LandingContentSe` or so? Just to slim this one down a bit and separate the responsibilities.

Reply:
I tried that but sending "textKeys" was a nightmare, I have no idea what type it is and don't want to implicitly cast it to any. I'll try it again though.

Reply:
I think you could put `const textKeys = useTextKeys()` inside these components also :)